packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,260 @@
+          <div className="ds-c-tooltip__content ds-base">
+            {children}
+            <div className="ds-u-justify-content--end ds-u-display--flex">
+              <Button
+                className="qa-tooltip-close-button"
+                size="small"
+                onClick={() => this.hideTooltip()}
+              >
+                Close
+              </Button>
+            </div>
+          </div>
+          <div
+            style={{ left: arrowProps.style.left }}
+            className="ds-c-tooltip__invisible-button"
+            onTouchStart={() => this.hideTooltip()}
+          />
+        </div>
+      </FocusTrap>
+    );
+
+    const nonInteractiveContent = (arrowProps, arrowStyle) => (
+      <Fragment>
+        <div className="ds-c-tooltip__arrow" ref={arrowProps.ref} style={arrowStyle} />
+        <div className="ds-c-tooltip__content ds-base">{children}</div>
+        <div
+          style={{ left: arrowProps.style.left }}
+          className="ds-c-tooltip__invisible-button"
+          onTouchStart={() => this.hideTooltip()}
+        />
+      </Fragment>
+    );
+
+    return ReactDOM.createPortal(
+      <Transition in={this.state.showTooltip} unmountOnExit timeout={transitionDuration}>
+        {(transitionState) => (
+          <Popper
+            positionFixed={positionFixed}
+            placement={placement}
+            modifiers={{ offset: { offset: TOOLTIP_OFFSET_OPT } }}
+          >
+            {({ placement, ref, style, arrowProps }) => {
+              // Need to add back 1/2 width of arrow to the left placement of the
+              // tooltip container so the arrow shows up at exactly 50% as the
+              // arrow container has a width/height we are setting that the
+              // arrowProps positioning here does not account for
+              let leftArrowOffset = 0;
+              if (parseInt(arrowProps.style.left, 10)) {
+                leftArrowOffset = arrowProps.style.left + 8;
+              }
+              const arrowStyle = { left: leftArrowOffset };
+
+              // Can't directly modify style, so copy and add styles from props
+              const newStyle = {
+                ...style,
+                ...transitionStyles[transitionState],
+                ...{
+                  maxWidth: tooltipMaxWidth,
+                  zIndex: tooltipZIndex,
+                },
+              };
+              return (
+                <div
+                  id={`tooltip-${id}`}
+                  ref={ref}
+                  className={classNames('ds-c-tooltip__container', tooltipBodyClassName, {
+                    'inverse-tooltip-body': inverse,
+                  })}
+                  style={newStyle}
+                  onMouseEnter={() => this.showTooltip()}
+                  onMouseLeave={() => this.hideTooltip()}
+                  data-placement={placement}
+                  modifiers={{ offset: TOOLTIP_OFFSET }}
```

Comment:
@hannaliebl do you know why this `TOOLTIP_OFFSET` is needed? I tried changing it's value and removing it from this `modifiers` and nothing seemed to change